### ISSUE TITLE
[Partner submission] subrecipient tab entity type not recognized

### ISSUE THREAD

**Issue by usdr-grants** [usdigitalresponse/arpa-reporter#402]
I think the new update to the subrecipient type caused an issue w/ the upload validations!
https://www.jotform.com/uploads/grants.dev/212695501572052/5329524122719271972/Screenshot%202022-07-06%20173249.png
https://www.jotform.com/uploads/grants.dev/212695501572052/5329524122719271972/Screenshot%202022-07-06%20173257.png

**Comment by jcomeau1** [comment]
The image links aren't working. I'll reach out to Jessica.

**Comment by jcomeau1** [comment]
Okay, I followed up with Jessica. It looks like it's just failing the validation in Render, which isn't a surprise since we haven't deployed yet. Here's her two images.

Also, I did ask her to try to confirm through the portal whether the "-" is actually needed.

![JessicaBug01](https://user-images.githubusercontent.com/78866741/177651176-5e3b277b-2bac-408c-8278-a22bf42afb61.png)
![JessicaBug02](https://user-images.githubusercontent.com/78866741/177651188-404d8d58-e379-4ca4-961c-c5d05b32d89f.png)
